pom.xml

```diff
@@ -264,8 +264,8 @@
     <version.org.sonatype.sisu>2.3.0</version.org.sonatype.sisu>
     <version.org.sonatype.sisu.sisu-guice>3.2.3</version.org.sonatype.sisu.sisu-guice>
     <version.org.springframework>5.2.9.RELEASE</version.org.springframework>
-    <version.org.springframework.boot>2.3.4.RELEASE</version.org.springframework.boot>
-    <version.org.springframework.amqpboot>2.3.4</version.org.springframework.amqpboot>
+    <version.org.springframework.boot>2.4.9</version.org.springframework.boot>
+    <version.org.springframework.amqpboot>2.3.6.redhat-00004</version.org.springframework.amqpboot>
```

Comment:
Not sure where is this used, and if it's the right version (not 2.4.x)

Reply:
Removed amqp-10-jms-spring-boot-starter as I didn't find any references in kie projects.

Reply:
do not use here any version with -redhat-#### suffix